<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeEmployee</class>
 <widget class="QDialog" name="ChangeEmployee">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>535</width>
    <height>706</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QRadioButton" name="radioButton_man">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>160</y>
     <width>44</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>男</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>40</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>员工信息</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_woman">
   <property name="geometry">
    <rect>
     <x>323</x>
     <y>160</y>
     <width>45</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>女</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_age">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>230</y>
     <width>89</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_jobTitile">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>420</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_edubkg">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>360</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_salary">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>550</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_dept">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>490</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_name">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>100</y>
     <width>142</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_phonenumber">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>290</y>
     <width>142</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>141</x>
     <y>91</y>
     <width>91</width>
     <height>501</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_num_2">
      <property name="text">
       <string>姓名：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_name_2">
      <property name="text">
       <string>性别：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_usable_2">
      <property name="text">
       <string>年龄：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_requir_2">
      <property name="text">
       <string>电话号码：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>学历:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_job_2">
      <property name="text">
       <string>职称:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>部门:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_salary_2">
      <property name="text">
       <string>工资:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
